.github/workflows/build-and-push-ecr.yml

```
@@ -101,6 +108,50 @@ jobs:
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v2
 
+      - name: Collect environment build args
+        id: collect-args
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DOCKER_BUILD_ARGS: ${{ inputs.docker_build_args }}
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
```

actionlint (1.7.11)

[error] property "sentry_auth_token" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; private_modules_token: string}

(expression)